### ISSUE TITLE
Separate repo for gCastle ?

### ISSUE THREAD

**Issue by impredicative** [huawei-noah/trustworthyAI#39]
Would it make sense to have gCastle in its own repo? Just wondering.

**Comment by shaido987** [comment]
Yes, we agree with you. This is something we are trying to push for internally but it concerns the open-source policy of our lab and may be hard to achieve in the near future.

**Comment by kelizhang** [comment]
we believe this issue will be well settled in the future!
